### PR TITLE
Persist state

### PR DESCRIPTION
**Pull request by Dan-Y-Ko**
I moved the dispatch to the see meals nearby button to test out the functionality of the state persistence. Confirmed only the location gets preserved so if the user gets an error that won't get persisted.

### CODE REVIEW

File: src/screens/Home.tsx [hunk 1]
```diff
@@ -1,66 +1,72 @@
...
+import { getLocationsAsync, useAppDispatch } from "@store";
 
 type Nav = {
   navigate: (value: string) => void;
-}
-  
+};
+
 export default function Home() {
+  const [searchText, setSearchText] = useState("");
```
Comment (lindentree) [inline src/screens/Home.tsx:13]: Shouldn’t the location state also live in the redux store?

Comment (Dan-Y-Ko) [reply src/screens/Home.tsx:13]: The location is in the store. 🤔Are we talking about different things here?

Comment (lindentree) [reply src/screens/Home.tsx:13]: Yes, you're right, my mistake. We'll need to refactor the search query hook to use the redux store in another ticket.

**Review by lindentree — APPROVED** [review]
LGTM

**Review by rweitzman — APPROVED** [review]

**Pull request merged**